CONTRIBUTING.md

```
@@ -48,19 +48,181 @@ Ask anything in our [Discord server](https://discord.com/invite/NAb6H3UTjK) —
+```bash
 npm run dev
 ```
 
-By default, the frontend expects the backend on `http://localhost:3001`. Update configs if needed.
+The client will run at `http://localhost:5173`.
+
+#### Step 5: Access the Application
+
+Open your browser and navigate to:
+
+- **Frontend**: http://localhost:5173
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)